spec/javascripts/health-care/components/personal-information/VeteranAddressSection.spec.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+
+import VeteranAddressSection from '../../../../../_health-care/_js/components/personal-information/VeteranAddressSection';
+
+describe('<VeteranAddressSection>', () => {
+  const nullAddress = [{
+    street: null,
+    city: null,
+    country: null,
+    state: null,
+    zipcode: null,
+  }];
+
+  it('Sanity check the component renders', () => {
+    const tree = SkinDeep.shallowRender(<VeteranAddressSection data={{ address: nullAddress }}/>);
+    const vdom = tree.getRenderOutput();
+    expect(vdom).to.have.property('type', 'fieldset');
```

Comment:
I'm a little concerned about checking for the type of highest level element as a check for whether or not the component renders. The fact that the highest level element in `VeteranAddressSection` is a `fieldset` isn't a deliberate choice, just a result of the particular markup within this section and not needing a wrapper div. Some sections first have a `div` because they might have multiple `fieldset`s and therefore need a wrapper `div`. I wonder if there is something more consistent across sections that we know each one should have? For example, each section should probably have at least one `h4`. Just feels like maybe we want our test to check for something that we know should be true about a section.